dev/docker-compose.yml

```diff
@@ -80,6 +80,39 @@ services:
       - pg
       - testdb
 
+  openssl:
+    image: svagi/openssl:latest
```

Comment:
do we want to use `latest`? as this is not something that we are testing maybe it will be better to have a static version.

Reply:
There's no other tag, sorry.
I looked for other images as well, but couldn't find anything with tag other than `latest`.